app/models/state_file_archived_intake_request.rb

```diff
@@ -0,0 +1,38 @@
+# == Schema Information
+#
+# Table name: state_file_archived_intake_requests
+#
+#  id                             :bigint           not null, primary key
+#  details                        :jsonb
```

Comment:
what goes into the `details`?

Reply:
🤔 this might have been a copy pasta mistake on my part. i think we can remove it.